bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1290 @@
+		<label>Set</label>
+		<description>Set</description>
+
+	</channel-type>
+
+	<channel-type id="onoff">
+		<item-type>Switch</item-type>
+		<label>On/Off</label>
+		<description>Switch on/off</description>
+	</channel-type>
+
+	<channel-type id="sleep">
+		<item-type>Switch</item-type>
+		<label>Set Timer</label>
+		<description>The output will be timed for nCYCLES of the Node associated timer</description>
+	</channel-type>
+
+	<channel-type id="autoMode">
+		<item-type>Switch</item-type>
+		<label>On/Off Auto Mode</label>
+		<description>Switch on/off</description>
+	</channel-type>
+
+	<channel-type id="stateOnOff">
+		<item-type>Contact</item-type>
+		<label>State On/Off</label>
+		<description>Light on/off</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+
+	<channel-type id="stateOpenClose">
+		<item-type>Contact</item-type>
+		<label>State Open/Closed</label>
+		<description>Contact Open/Closed</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="rollerShutter">
+		<item-type>RollerShutter</item-type>
+		<label>Rollershutter</label>
+		<description>Rollershutter Up/Down</description>
+		<category>Blinds</category>
+		<state min="0" max="100" pattern="%d %%"/>
+	</channel-type>
+
+	<channel-type id="rollerShutterState">
+		<item-type>String</item-type>
+		<label>State</label>
+		<description>State of rollershutter/windows</description>
+		<state readOnly="true">
+			<options>
+				<option value="opening">Opening</option>
+				<option value="stop">Stop</option>
+				<option value="closing">Closing</option>
+				<option value="limSwitchOpen">Limit Switch Open</option>
+				<option value="limSwitchClose">Limit Switch Close</option>
+				<option value="stateOpen">Opened</option>
+				<option value="stateClose">Closed</option>
+				<option value="NoLimSwitch">No Limit Switch</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="lastMessage" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Last Message</label>
+		<description>Last Message emitted by the module</description>
+		<category>QualityOfService</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="lastStatusStored" advanced="false">
+		<item-type>DateTime</item-type>
+		<label>Last Status Stored</label>
+		<description>Last Status Store</description>
+		<category>QualityOfService</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="healthy">
+		<item-type>Number</item-type>
+		<label>Healthy</label>
+		<description>Souliss Healthy</description>
+		<tags>
+			<tag>QualityOfService</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="float">
+		<item-type>Number</item-type>
+		<label>Value</label>
+		<description>Floating Point Input</description>
+		<tags>
+			<tag>sensors</tag>
+		</tags>
+		<state readOnly="true" pattern="%.1f "/>
+	</channel-type>
+
+	<channel-type id="float6n">
+		<item-type>Number</item-type>
```

Comment:
Use UoM. See above.
```suggestion
		<item-type>Number:Temperature</item-type>
```

Reply:
It could be various types:
https://github.com/souliss/souliss/wiki/Typicals

So i'll prefer leave them as is, otherwise must define many other channels.. since they are little used, worth to do the work ?

Reply:
OK